blog/src/Auth/Controller/ChangePasswordController.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Auth\Controller;
+
+use App\Auth\AuthService;
+use App\Auth\Identity;
+use App\Auth\IdentityRepository;
+use App\Auth\Form\ChangePasswordForm;
+use App\Service\WebControllerService;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Yiisoft\FormModel\FormHydrator;
+use Yiisoft\Http\Method;
+use Yiisoft\Session\SessionInterface as Session;
+use Yiisoft\Session\Flash\Flash;
+use Yiisoft\Translator\TranslatorInterface as Translator;
+use Yiisoft\User\CurrentUser;
+use Yiisoft\Yii\View\ViewRenderer;
+
+final class ChangePasswordController
+{
+    public function __construct(
+      private Session $session,
+      private Flash $flash,
+      private Translator $translator,
+      private CurrentUser $currentUser,
+      private WebControllerService $webService, 
+      private ViewRenderer $viewRenderer,
+    )
+    {
+      $this->currentUser = $currentUser;
+      $this->session = $session;      
+      $this->flash = new Flash($session);
+      $this->translator = $translator;
+      $this->viewRenderer = $viewRenderer->withControllerName('changepassword');
```

Comment:
```suggestion  
      $this->flash = new Flash($session);
      $this->viewRenderer = $viewRenderer->withControllerName('changepassword');
```

Reply:
Get the following Psalm Level 1 errors: 
![image](https://github.com/user-attachments/assets/62d081eb-702e-4f85-ab09-7928b3605652)

Reply:
With the code as it is I am getting no Psalm Level 1 errors. 

rossaddison/invoice has the following psalm.xml file which incorporates the following: 

````
<plugins><pluginClass class="Psalm\SymfonyPsalmPlugin\Plugin"/></plugins>    
    <projectFiles>
        <directory name="config" />
        <directory name="resources/views" />
        <directory name="src" />
        <file name="public/index.php"/>
        <file name="yii"/>
        <file name="autoload.php"/>
        <ignoreFiles>
            <directory name="vendor/yiisoft/requirements/src" />
        </ignoreFiles>        
    </projectFiles>    
</psalm>
````

Reply:
The config directory is included.